src/features/semantic_tokens.zig

```diff
@@ -51,14 +51,13 @@ pub const TokenModifiers = packed struct(u16) {
     static: bool = false,
     deprecated: bool = false,
     abstract: bool = false,
-    @"async": bool = false,
```

Comment:
This is a [semanticTokenModifiers](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#semanticTokenModifiers) in the LSP specification and not Zig's async. This shouldn't be removed.

Reply:
Thank you for the catch. Fixed in https://github.com/zigtools/zls/pull/2394/commits/8ede2b52ea331befb3511ed26f033dbd146fcf6d.